### ISSUE TITLE
feature request: selectable download path for dcc

### ISSUE THREAD

**Issue by stolendata** [psychs/limechat#106]
As title says, would be great to get a configuration slot to enter the path where incoming files via DCC will end up, instead of them being forced to ~/Downloads/.

**Comment by foice** [comment]
I agree that's a desirable feature

**Comment by iruy** [comment]
In 2017, this is still a must have feature :)

**Comment by iruy** [comment]
I'd say the above for the 2018 and 2019 too..

**Comment by witedragen** [comment]
Yes sir, 2019 (and almost 2020) this is still a must have feature 😸

**Comment by davidlondono** [comment]
now on 2020 and feature not on the app

**Comment by pcace** [comment]
now on 2021 and feature not on the app

**Comment by knutwurst** [comment]
2023 and I still can not switch the download folder. Great.

**Comment by Schariez** [comment]
I would love to choose the folder...

**Comment by greenbriel** [comment]
Thirteen years later I think we can assume this isn't happening! Too bad.
